CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 ### Bugfixes
 
 * Lorem ipsum.
+### Internals
+
+* Implemented inter-process CondVars on Windows (Win32 + UWP). They should be fair and
+  robust.
```

Comment:
Please use the existing `### Internals` section ~20 lines below this. A PR should generally only insert a single item (i.e. no headers).